src/index.ts

```diff
@@ -1,120 +1,170 @@
-  * @param {string} key - the key which was used to save the locale in the cache.
-  * @returns {Promise<string>} - A Promise that when resolved, returns the json serialized locale.
-  */
-  read(key: string): Promise<string>;
-
-  /**
-  * Writes a locale from cache.
-  * @param {string} key - the key which will be used to save the locale in the cache.
-  * @param {string} value - The json serialized locale.
-  * @returns {Promise<string>} - A Promise that when resolved, indicates that the locale was stored.
-  */
-  write(key: string, value: string): Promise<void>;
+    /**
+     * Reads a locale from cache.
+     * @param {string} key - the key which was used to save the locale in the cache.
+     * @returns {Promise<string>} - A Promise that when resolved, returns the json serialized locale.
+     */
+    read(key: string): Promise<string>;
+
+    /**
+     * Writes a locale from cache.
+     * @param {string} key - the key which will be used to save the locale in the cache.
+     * @param {string} value - The json serialized locale.
+     * @returns {Promise<string>} - A Promise that when resolved, indicates that the locale was stored.
+     */
+    write(key: string, value: string): Promise<void>;
 }
 
 export interface CalingaBackendOptions {
-  /**
-   * The version of the Calinga project
-   */
-  version?: string;
-  /**
-   * The base URL of the Calinga service. Should not be changed.
-   */
-  serviceBaseUrl?: string;
-  /**
-   * A cache to store locales that were returned from the Calinga service
-   */
-  cache?: Cache;
-  /**
-   * Preshipped translations, similar to i18next's options.resources.
-   */
-  resources?: Resource;
+    /*
+     * The name of the calinga project
+     */
+    project: string;
+    /*
+     * The version of the Calinga project
+     */
+    version?: string;
+    /*
+     * The base URL of the Calinga service. Should not be changed.
+     */
+    serviceBaseUrl?: string;
+    /*
+     * A cache to store locales that were returned from the Calinga service
+     */
+    cache?: Cache;
+    /*
+     * Preshipped translations, similar to i18next's options.resources.
+     */
+    resources?: Resource;
+
+    /*
+     * Adds a development language if set to 'true'
+     */
+    devMode?: boolean;
 }
 
 export class CalingaBackend implements BackendModule<CalingaBackendOptions> {
-  static type = 'backend';
-  type: 'backend';
+    static type = 'backend';
+    type: 'backend';
 
-  services: Services;
-  options: CalingaBackendOptions;
+    services: Services;
+    options: CalingaBackendOptions;
 
-  loadPath = 'translations/{{project}}/{{version}}/{{language}}';
+    loadPath = 'translations/{{project}}/{{version}}/{{language}}';
+    localesPath = 'locales/{{project}}/{{version}}';
 
-  constructor(services: Services, options: CalingaBackendOptions) {
-    this.init(services, options);
-  }
+    static languages: string[];
+    static onLanguagesChanged: (languages: string[]) => void;
```

Comment:
We should add some hints to our readme file on how to access the list of available languages...